### DISCUSSION TITLE
Prevent direct access with rewrites

### DISCUSSION THREAD

**Discussion by isshesure** [vercel/vercel#7629] [Help]
Hi, if you have a vercel/nextjs rewrite configured to allow `foo.domain.com` to be accessed through `www.domain.com/foo`, is there also a way to prevent direct access to `foo.domain.com` ?

**Comment by leerob** [comment]
Hey! You could set up a redirect from `foo.domain.com` to `domain.com` as well.

**Reply by isshesure** [reply]
Thanks, could this also be configured for deep links (all could direct to `www.domain.com`)

For example `foo.domain.com/foo`

Current setup:

```
async redirects() {
  return [
    {
      source: "/:path*",
      destination: "https://www.domain.com",
      permanent: true,
    }
  ];
}
```
